core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```
@@ -103,6 +102,21 @@ object ConsumerGroupCommand extends Logging {
           case _ => throw new IllegalStateException(s"delete is not supported for $consumerGroupService.")
         }
       }
+      else if (opts.options.has(opts.resetOffsetsOpt)) {
+        consumerGroupService match {
```

Maybe it would be simpler to add `resetOffsets` to `ConsumerGroupService` and let it throw `UnsupportedOperationException` for the default implementation. Then we don't need to pattern match here.

---

Agree. It is fixed now.